### PR TITLE
Checking migration to NG16

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/auth/guard/auth-guard-sso-role.service.ts [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
+import { ActivatedRouteSnapshot, Router } from '@angular/router';
 import { MatDialog } from '@angular/material/dialog';
 import { UserAccessService } from '../services/user-access.service';
 
 @Injectable({
     providedIn: 'root'
 })
-export class AuthGuardSsoRoleService implements CanActivate {
+export class AuthGuardSsoRoleService  {
```
Comment (DenysVuika) [inline lib/core/src/lib/auth/guard/auth-guard-sso-role.service.ts:26]: is this something the angular upgraade does? we don't need CanActivate anymore?

File: lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss
Comment (DenysVuika) [inline lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss:1]: we hav a big PR reworking property panel, so many these classes will go away

File: lib/core/src/lib/auth/guard/auth-guard-sso-role.service.ts (hunk 1 above)
Comment (VitoAlbano) [reply lib/core/src/lib/auth/guard/auth-guard-sso-role.service.ts:26]: it does seem so i haven't changed anything in the code in this stage, just checking the upgrade

**Review by mauriziovitale — APPROVED** [review]

File: lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss
Comment (VitoAlbano) [reply lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss:1]: probably but the upgrate will still be a big PR but i have an idea to have it a little bit smaller so i'll give it a shot and i will wrap up this and the other one

**Pull request closed without merging**